pkg/package-server/apis/operators/v1/packagemanifest_types.go

```diff
@@ -10,10 +10,10 @@ import (
 // PackageManifestList is a list of PackageManifest objects.
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 type PackageManifestList struct {
-	metav1.TypeMeta
-	metav1.ListMeta
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
```

Comment:
We don't use `omitempty` on the `ListMeta` for our other API structs, not sure if this will cause problems.